app/locales/pt-BR.json

```diff
@@ -0,0 +1,118 @@
+    "home_unknown_header": "Desconhecido",
+    "home_unknown_subtext": "Não podemos dizer se você está em risco, a menos que você permita o aplicativo acessar sua localização.",
+    "home_setting_off_header": "Desconhecido",
+    "home_setting_off_subtext": "Não podemos dizer se você está em risco, a menos que você ative seu histórico de localização na tela de configurações.",
+    "import_step_1": "1. Realize login na sua conta do Google e baixe o seu histórico de localização",
+    "import_step_2": "2. Depois que você baixar, abra esta tela novamente. Os dados serão importados automaticamente.",
+    "import_title": "Importar localização",
+    "latest_news": "Últimas notícias",
+    "launch_done_header": "Tudo pronto",
+    "launch_done_subheader": "Você está pronto para seguir. Lembre-se, você sempre pode alterar suas preferências depois. ",
+    "launch_enable_location": "Ativar localização",
+    "launch_enable_notif": "Ativar notificações",
+    "launch_finish_set_up": "Concluir configurações",
+    "launch_get_started": "Iniciar",
+    "launch_location_access": "Acesso à localização",
+    "launch_location_header": "Para lembrar onde você deve ir, seu celular precisa ter sua localização",
+    "launch_location_subheader": "Não se preocupe, as informações nunca saem do seu dispositivo, a menos que você decida explicitamente que deseja compartilhá-las.",
+    "launch_next": "Próximo",
+    "launch_notif_header": "Você receberá uma notificação se cruzar o caminho com uma pessoa infectada.",
+    "launch_notif_subheader": "Não iremos incomodá-lo, exceto para compartilhar atualizações sobre seus potenciais riscos de exposição.",
+    "launch_notification_access": "Permitir notificações",
+    "launch_screen1_header": "O caminho de volta ao normal começa aqui.",
+    "launch_screen2_header": "Seja notificado se você encontrar alguém diagnosticado para COVID-19.",
+    "launch_screen2_subheader": "Conhecimento é poder.",
+    "launch_screen3_header": "Se seu teste para COVID-19 deu positivo, você pode escolher enviar os seus dados anonimamente",
+    "launch_screen3_subheader": "Que ajuda a manter sua comunidade inteira salva.",
+    "launch_screen4_header": "Você tem o controle completo. Informações são salvas apenas no seu celular ",
+    "launch_screen4_subheader": "Se seu teste para COVID-19 deu positivo, você sozinho pode optar por compartilhar.",
+    "launch_set_up_phone": "Configurar meu celular",
+    "legal_page_title": "Legal",
+    "less_than_one_minute": "menos de um minuto",
+    "loading_public_data": "carregando dados...",
+    "location_disabled_message": "COVID Safe Paths requer serviços de localização.",
+    "location_disabled_title": "Serviço de localização desativado",
+    "location_enabled_message": "COVID Safe Paths está armazenando de forma segura suas coordenadas GPS a cada cinco minutos neste dispositivo.",
```

Comment:
a cada cinco minutos or
uma vez a cada cinco minutos?

The English line is: once every five minutes

Reply:
Both have the same meaning in Portuguese, but the way you put it makes it more explicit. 

Soon I will send corrections taking into account all your suggestions. Sorry for the mistakes.

Reply:
no worries at all and no need to apologize. The review is for figuring out what works best and you can take whatever suggestions that you feel are appropriate :)